chatterbot/adapters/plugins/evaluate_mathematically.py

```diff
@@ -0,0 +1,200 @@
+from .plugin import PluginAdapter
+import re
+import os, json
+
+class EvaluateMathematically(PluginAdapter):
+
+    def should_answer(self, input_text):
+        """
+        Determines whether it is appropriate for this plugin
+        to respond to the user input.
+        """
+
+        response = self.process( input_text )
+
+        if response == False:
+            return False
+        else:
+            return True
+
+
+    def process(self, input_text):
+        """
+        Takes a statement string.
+        Returns the simplified statement string
+        with the mathematical terms "solved".
+        """
+
+        # Getting the mathematical terms within the input statement
+        expression, string = self.simplify_chunks( self.normalize( input_text ) )
+
+        # Returning important information
+        try:
+            string += '= ' + str( eval( string ) )#self.evaluate( string ) )
+
+            return string
+        except:
+            return False
+
+
+    def simplify_chunks(self, input_text):
+        """
+        Separates the incoming text.
+        """
+
+        expression = []
+        string = ''
+
+        for chunk in input_text.split( ' ' ):
+
+            is_integer = self.isInteger( chunk )
+
+            if is_integer == False:
```

Comment:
In Python you should use `if x is False` to check for `False` instead of using `==`.

```python
>>> 0 == False
True
>>> 0 is False
False
```